spec/requests/graphql/mutations/create_habit_entry_spec.rb

```diff
@@ -0,0 +1,59 @@
+require "rails_helper"
+
+describe 'create habit entry' do
+
+  before :each do
+    @user = create(:user)
+    @habit = create(:habit)
+    create(:habit, id: 7)
+    create(:habit, id: 13)
+  end
+
+  it 'can create a habit entry' do
+    post '/graphql', params: { query: query }
+
+    expect(@user.habit_entries.count).to eq(3)
+    expect(@user.habit_entries.first.status).to eq(1)
+    expect(@habit.habit_entries.first.status).to eq(0)
+  end
+
+  it 'returns errors when given bad query' do
+    post '/graphql', params: { query: bad_query }
+
+    parsed = JSON.parse(response.body, symbolize_names: true)
+
+    expect(parsed).to have_key(:errors)
+  end
+
+  def query
+    <<~GQL
+    mutation {
+      createHabitEntry(
+        input: {
+          params: [{id: 13}, {id: 7}]
```

Comment:
This changed from our earlier conversation. Now its just habit ids in a hash/object in an array.